### DISCUSSION TITLE
do you need Xformers?

### DISCUSSION THREAD

**Discussion by hoodady** [Panchovix/stable-diffusion-webui-reForge#39] [Q&A]
--xformers --always-gpu --disable-nan-check -cuda-malloc --cuda-stream --pin-shared-memory i saw this command was recommended, do you need Xformers for forge?

**Comment by Panchovix** [comment]
Not always needed, if you disable xformers you can use sdpa

**Reply by hoodady** [reply]
what is sdpa?

**Reply by Panchovix** [reply]
Another way to do cross optimization included in torch itself.
